lib/oci_utils/migrate/exception.py

```diff
@@ -0,0 +1,68 @@
+# oci-utils
+#
+# Copyright (c) 2019, 2020 Oracle and/or its affiliates. All rights reserved.
+# Licensed under the Universal Permissive License v 1.0 as shown
+# at http://oss.oracle.com/licenses/upl.
+
+""" Module with oci migrate related exceptions.
+"""
+
+
+class OciMigrateException(Exception):
+    """ Exceptions for the Image Migrate to OCI context.
+    """
+    __args = None
+
+    def __init__(self, message=None):
+        """
+        Initialisation of the Oci Migrate Exception.
+
+        Parameters
+        ----------
+        message: str
+            The exception message.
+        """
+        self._message = message
+        assert (self._message is not None), 'No exception message given'
+
+        if self._message is None:
+            self._message = 'An exception occurred, no further information'
+
+    def __str__(self):
+        """
+        Get this OCISDKError representation.
+
+        Returns
+        -------
+        str
+            The error message.
+        """
+        return str(self._message)
+
+
+class NoSuchCommand(OciMigrateException):
+    """ Exception for command not found.
+    """
+    def __init__(self, command):
```

Comment:
This two methods should be removed as they do not actually overwrite the behavior of methods in upper class

Reply:
did not want to reuse the OCISDKError, and for 'future use'

Reply:
My  point is about NoSuchCommand class  which do not change the behavior off OciMigrateException methods and as such should be left empty

Reply:
yes, you are correct, will change this; (the 'future' use did not seem to be necessary)